experiment_buddy/experiment_buddy.py

```diff
@@ -223,7 +221,11 @@ def _ask_experiment_id(cluster, sweep):
         experiment_id = tkinter.simpledialog.askstring(title, "experiment_id")
         root.destroy()
     except:
-        experiment_id = input(f"Running on {title}\ndescribe your experiment (experiment_id):\n")
+        if os.environ['BUDDY_CURRENT_TESTING_BRANCH']:
```

Comment:
os.environ.get ?
what happens if the var is not set?

Reply:
Explodes, of course :D
Using `.get` instead